diff --git a/themes/donal.zsh-theme b/themes/donal.zsh-theme
new file mode 100644
index 0000000..a188e71
--- /dev/null
+++ b/themes/donal.zsh-theme
@@ -0,0 +1,20 @@
+source /Users/doevans/workspace/zsh-git-prompt/zshrc.sh
+
+#use extended color palette if available
+if [[ $terminfo[colors] -ge 256 ]]; then
+    turquoise="%F{81}"
+    orange="%F{166}"
+    purple="%F{135}"
+    hotpink="%F{161}"
+    limegreen="%F{118}"
+else
+    turquoise="%F{cyan}"
+    orange="%F{yellow}"
+    purple="%F{magenta}"
+    hotpink="%F{red}"
+    limegreen="%F{green}"
+fi
+
+PROMPT=$'
+%{$purple%}%n%f at %{$orange%}%m%f in %{$limegreen%}%~%f
+$(git_super_status) $ '
diff --git a/themes/steeef.zsh-theme b/themes/steeef.zsh-theme
index 3532d3b..93c44a6 100644
--- a/themes/steeef.zsh-theme
+++ b/themes/steeef.zsh-theme
@@ -82,7 +82,7 @@ function steeef_chpwd {
 add-zsh-hook chpwd steeef_chpwd
 
 function steeef_precmd {
-    if [[ -n "$PR_GIT_UPDATE" ]] ; then
+   if [[ -n "$PR_GIT_UPDATE" ]] ; then
         # check for untracked files or updated submodules, since vcs_info doesn't
         if git ls-files --other --exclude-standard 2> /dev/null | grep -q "."; then
             PR_GIT_UPDATE=1
@@ -94,7 +94,7 @@ function steeef_precmd {
 
         vcs_info 'prompt'
         PR_GIT_UPDATE=
-    fi
+   fi
 }
 add-zsh-hook precmd steeef_precmd
 
